docs/blockchain/contracts/nft-contract/nft-actions/mkfctofr.a.experimental.md

```diff
@@ -0,0 +1,74 @@
+---
+title: 'mkfctofr.a'
+order: 40
+
+---
+
+# mkfctofr.a
+
+Make a buy offer on a given factory.
+
+## Technical Behavior
+
+The offer should be done at the time which is in the range of trading window of the factory.
+
+The action stores the offer to `fctoffer.a` table with the specified arguments. The new offer ID is read from `next.fctofr` table whose value field is then incremented.
```

Comment:
... value field is then incremented
should be
... `value` field is then incremented

Reply:
updated